react-events/my-app/src/ImageCaption.tsx

```diff
@@ -0,0 +1,12 @@
+type Props = {
+  imageCap: string[];
```

Comment:
The best way to do this that makes the component most reusable is to just take a single string as the value. Then in the parent you do the indexing.